jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/Importer.java

```diff
@@ -7,305 +7,408 @@
 import java.io.IOException;

```

Comment:
Importer class should ideally be parameterized so that it can be used for any inbox folder. There are going to be many different types of file that we will need to import. If we keep making this class bigger and bigger each time a new file is introduced, it will become humongous. The object of this class needs to be instantiated with a location path and an object to process the files that are detected in that location. Then we can instatiate an object of Importer class for pure BSMs, one for MessageFrames, one for TIMs, etc, etc. Again, let's refactor it for next sprint if we don't have time to do it in the current sprint.